packages/noya-state/src/selectors.ts

```diff
@@ -26,6 +26,10 @@ export const getCurrentPageIndex = (state: ApplicationState) => {
   return pageIndex;
 };
 
+export const getSharedSwatches = (state: ApplicationState) => {
+  return state.sketch.document.sharedSwatches ?? [];
```

Comment:
From our call, we can probably return a `Swatch[]` instead of `SwatchContainer`:

```ts
export const getSharedSwatches = (
  state: ApplicationState,
): Sketch.Swatch[] => {
  return (
    state.sketch.document.sharedSwatches?.objects ?? []
  );
};
```

Reply:
I updated this!